website/docs/reference/widgets/list.md

```diff
@@ -115,10 +149,21 @@ Properties allow you to customize the widget, connect it to other widgets and tr
 | Property                   | Description                                                                                                                                                                                                                                               |
 | -------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | **Items**                  | Allows you to bind static or dynamic data collection to the widget.                                                                                                                                                                                       |
```

Comment:
Can we also write that it only accepts arrays here as well?

Reply:
updated